File: CSharp/TestExercisesKatas/TestDoublesExercise/Bank/Transaction.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TestExercises.Bank
{
    public class Transaction
    {
        public Money Amount { get; }
        public DateTime TransactionDate { get; }

        public Transaction(Money value, DateTime transactionDate)
        {
            Amount = value;
            TransactionDate = transactionDate;
        }
    }
}

File: Java/src/main/java/at/nagarro/exercises/bank/Transaction.java
package at.nagarro.exercises.bank;

import java.time.LocalDate;

public class Transaction {
    public Money amount;
    public LocalDate transactionDate;

    public Transaction(Money value, LocalDate transactionDate) {
        this.amount = value;
        this.transactionDate = transactionDate;
    }
}